manual_user_install.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/bash
+
+# tested with a clean 64 bit installation of Raspbian-Lite
+
+function install_core (){
+    echo "Installing OVOS core"
+    echo
+    pip3 install -U pip
+
+    # install ovos-core - ovos-backend-client has conflicting dependencies
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-backend-client
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-core
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-audio
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-ocp-audio-plugin
+    pip3 install tornado
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-messagebus
+    pip3 install adapt-parser
+    pip3 install padacioso
+
+    # dinkum listener
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-dinkum-listener
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-vad-plugin-webrtcvad
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-vad-plugin-silero
+
+    #Precise-lite wake-word cluster
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-ww-plugin-precise-lite
+    python3 -m pip3 install --upgrade setuptools
+    # if pip3 install tflite_runtime is problematic try:
+    # we install full tensorflow (because it just works, takes a while to install)
+    # sudo apt install -y libatlas-base-dev
+    #  pip3 install tensorflow --no-cache-dir
+    pip3 install tflite_runtime
+    pip3 install PyYAML
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-workshop
+    sudo apt install -y python3-fann2
+    pip3 install padatious
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-lingua-franca
+    pip3 install PyAudio
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-ww-plugin-pocketsphinx
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-ww-plugin-precise
```

Comment:
oly need precise-lite, its one or the other

Reply:
Cool.  Which should I keep? I'm not qualified to make the choice.

Reply:
Lite would be better

Reply:
sphynx now commented out in my version. will test.